.kuzzlerc.sample

```diff
@@ -19,7 +19,17 @@
     // (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS)
     "accessControlAllowOrigin": "*",
     "accessControlAllowMethods": "GET,POST,PUT,DELETE,OPTIONS,HEAD",
-    "accessControlAllowHeaders": "Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With, Content-Encoding, Content-Length, X-Kuzzle-Volatile"
+    "accessControlAllowHeaders": "Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With, Content-Encoding, Content-Length, X-Kuzzle-Volatile",
+    
+    // [Swagger]
+    //   * enabled: 
+    //      enable the Swagger specifications endpoint
+    //   * yamlOutput:
+    //      use YAML format to print Swagger specifications instead of JSON
+    "swagger": {
+      "enabled": true,
+      "yamlOutput": false
```

Comment:
Personal preference here, to be debated: what about an `output` property taking either "json" or "yaml" as values?

```js
      "output": "json"
```

This would need an assertion check when using the configuration, but otherwise I find that easier to read/understand. Here, one could assume that this enables YAML output on top of the JSON one.
WDYT?